### ISSUE TITLE
asciidoctor-mallard is not published in rubygems.org

### ISSUE THREAD

**Issue by dhanvi** [bk2204/asciidoctor-mallard#17]
As mentioned in the readme file asciidoctor-mallard is not published in the rubygems.org website.

https://rubygems.org/search?utf8=%E2%9C%93&query=asciidoctor-mallard

**Comment by mojavelinux** [comment]
Correct. We need to get a few more things finished, then we'll shoot for a first release.
